scripts/test_all_rpc.py

```
@@ -71,15 +71,16 @@ def test_frost_commit(ser):
         "message": "00" * 32
     }
     resp = send_request(ser, "frost_commit", params)
     if resp and "result" in resp:
         result = resp['result']
-        if "session_id" in result:
-            print(f"    PASS: session_id generated, commitment received")
+        if "commitment" in result:
+            print(f"    PASS: commitment received, index={result.get('index')}")
             return result
-        print(f"    FAIL: no session_id in response")
+        print(f"    FAIL: no commitment in response")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)